### ISSUE TITLE
Update VAMC Facility template to adopt V3 accordion component

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#16963]
## Description
VAMC Facility pages need the following updates to components:

Example pages:

- [Jamaica Plain VA Medical Center](https://www.va.gov/boston-health-care/locations/jamaica-plain-va-medical-center/)
- [Minneapolis VA Medical Center](https://www.va.gov/minneapolis-health-care/locations/minneapolis-va-medical-center/)

### V1 components that need to be upgraded (Actionable)
1. Accordion
   - Accordion sections (Prepare for your visit and services) are using the v1 component and need to be updated [to v3](https://design.va.gov/components/accordion#examples---v3). 

### Not in scope
* #17654

## Acceptance Criteria
- [x] Accordion component has been updated to v3
- [x] Update and run existing test(s).
- [ ] Presentation is verified the same across all supported browsers on Mac & Win
    - [ ] Chrome 
    - [ ] Firefox
    - [ ] Safari
    - [ ] Edge
    - [ ] Android
    - [ ] iOS
- [ ] Requires accessibility review

**Comment by maxx1128** [comment]
I estimate this ticket at a **3** for moving over a good amount of relatively simple elements over to components.

**Comment by jilladams** [comment]
TO DO: @jilladams split out V1 work to new ticket

**Comment by eselkin** [comment]
Should this also change: https://va.gov/boston-health-care/health-services/?

**Comment by jilladams** [comment]
Due to #17807 popping up, where adoption of components ran into some browser-specific styling issues, I've added ACs here to cover cross-browser verification.

**Comment by laflannery** [comment]
**FF Windows**
![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/e191dbfc-0a40-494e-b162-d297d18f892c)

**Edge** (I didn't put the fancy bookmarklet on Edge but confirmed these are the v3 variant)
![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/7d7b2cca-2731-44f0-a672-fa8f9b36fbe2)

**Chrome Windows**
![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/67473973-b0ff-4602-bbeb-3779e1714902)

_Will do tomorrow morning_
**FF Mac**

**Chrome Mac**

**Safari**

Android and iOS can't be tested until Staging/Prod unless we use ngrok @jilladams thoughts?

**Comment by jilladams** [comment]
Thoughts: https://github.com/department-of-veterans-affairs/va.gov-cms/issues/16997#issuecomment-2050641295
tl;dr: let's use mobile emulator to test lower envs on desktop, and verify on phones in prod.

**Comment by laflannery** [comment]
Confirmed on Prod:
![Screenshot 2024-04-15 at 8 39 10 AM](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/04d3af7b-d0ad-4f52-aaaa-2779639d13cd)
